src/docs/releases/0.0.1.md

```diff
@@ -0,0 +1,3 @@
+# Orchard Core Commerce 0.0.1
```

Comment:
Is this file auto-generated? If yes, how do we update _releases_ in the future? Please link a resource in the root readme.

Reply:
Manually created, as in OC.
Creating a release in GitHub will generate the PR merged since the latest release.
After first release (starting at the second), a link to compare the two releases could be included.

Change in .md, .yml files should not trigger the build.

We can document the release process as in https://docs.orchardcore.net/en/latest/docs/topics/publishing-releases/

Reply:
I see. I'm sure we'll streamline this after the MVP is done. Until then I don't anticipate new releases (this v0.0.1 was just to test the NuGet publishing workflow) so we can leave it as-is.